app/forms/questionnaire_form.py

```diff
@@ -386,8 +386,27 @@ def _get_period_limits(
         return minimum, maximum
 
     def _get_formatted_calculation_values(
-        self, answers_list: Sequence[str]
+        self, answers_sequence: Sequence[str]
     ) -> list[str]:
+        answers_list: list[str] = []
+        block_id = self.location.block_id if self.location else None
+        if block_id and self.schema.block_has_dynamic_answer(block_id):
+            list_name = (
+                self.schema.get_list_name_for_dynamic_answer(block_id)
+                if block_id
```

Comment:
Do you need the if here given we check for block above?

Reply:
My bad, now I can see it, seems like we don't need this at all.